docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -843,6 +857,21 @@ This could be useful to simplify the template structure:
 <1> The `name` will be resolved against the `item.parent`.
 <2> The `description` will be also resolved against the `item.parent`.
 
+[IMPORTANT]
```

Comment:
@angelozerr Let's start with the docs before we deprecate the `with` section ;-).

Reply:
Ok thanks for the update.